notebooks/book2/03/hierarchical_binom_rats.ipynb

```diff
@@ -0,0 +1,1831 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import jax
</pre><p>Add link to pymc version as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>